src/store/state/state-model.ts

```diff
@@ -8,123 +7,8 @@ import {
   EmptyNaics,
   StateModelIF
 } from '@/interfaces'
-import { AmlRoles, AmlTypes } from '@/enums'
 import { EmptyAddress } from '@bcrs-shared-components/interfaces'
 import { cloneDeep } from 'lodash'
-import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
-
-// const AMALGAMATING_BUSINESSES: AmalgamatingBusinessIF[] = [
```

Comment:
I moved this to the filing template mixin so it can be automatically added to whatever was already in the draft.

Now, THIS file has no tech debt.